tests/test_gui_rtdc.py

```diff
@@ -0,0 +1,94 @@
+from PyQt6 import QtCore, QtWidgets
+import pathlib
+import tempfile as tf
+import shutil
+
+from mpl_data_cast.gui.main import MPLDataCast
+from helper import retrieve_data
+
+data_path = pathlib.Path(__file__).resolve().parent / "data"
+
+
+def test_setting_paths(qtbot, tmp_path):
+    """Check that setting the input and output paths works."""
+    mpldc = MPLDataCast()
+    path_in = retrieve_data("rcp_rtdc_mask-contour_2018.zip")
+    path_out = tmp_path
+    mpldc.widget_input.lineEdit_input.setText(str(path_in))
+    mpldc.widget_output.lineEdit_output.setText(str(path_out))
+
```

Comment:
Just one more thing: Please test for something here (e.g. whether the tree is populated).

Reply:
Sorry, seems like I wrote only half a test there. I added some assert statements now.